packages/webpack5/src/helpers/flavor.ts

```diff
@@ -2,6 +2,23 @@ import { defaultConfigs } from '@nativescript/webpack';
 import { getAllDependencies } from './dependencies';
 import { error } from './log';
 
+/**
+ * Utility to determine the project flavor based on installed dependencies
+ * (vue, angular, react, svelete, typescript, javascript...)
+ */
+export function projectUsesCustomFlavor(): boolean {
+	const dependencies = getAllDependencies();
+	
```

Comment:
Could be simplified to something like:
```suggestion
	return [
	  'vue', 
	  'angular',
	  'react',
	  'svelte'
	].includes(determineProjectFlavor())
```